dashboard/app/models/contact_rollups_processed.rb

```diff
@@ -19,48 +19,52 @@ class ContactRollupsProcessed < ApplicationRecord
   DEFAULT_BATCH_SIZE = 10000
 
   # Aggregates data from contact_rollups_raw table and saves the results, one row per email.
-  #
   # @param [Integer] batch_size number of records to save per INSERT statement.
   def self.import_from_raw_table(batch_size = DEFAULT_BATCH_SIZE)
+    # Process the aggregated data row by row and save the results to DB in batches.
```

Comment:
There are no logic changes in this file. Just move the query builder into a separate method.